.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright 2007-present the original author or authors.
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I am not sure whether can change license of `MavenWrapperDownloader`, it is not source code of ShardingSphere.
Could you contact skywalking community to confirm it?

Reply:
I found from the code of `skywalking` that this generation purchase is also licensed under this license. It seems that it cannot be changed. I will ignore the `.mvn` for now.

Reply:
Oh, this was on their original header, even it is wrong. But we don't know what should we change to. And all things not updated after that(2+ years ago)

Reply:
Welcome to fix this on SkyWalking site too. This was deliberate. @totalo

Reply:
Okay, I will fix this question.